vnext/Microsoft.ReactNative/IReactDispatcher.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "IReactDispatcher.h"
+#include "ReactDispatcherHelper.g.cpp"
+
+using namespace winrt;
+using namespace Windows::Foundation;
+
+namespace winrt::Microsoft::ReactNative {
```

Comment:
Why prefix these namespaces `winrt::`?

Reply:
The `winrt::` prefix is commonly used for the C++/WinRT components. It is used by default by the C++/WinRT code generator. It is much easier to author our own components in that namespace. Plus, we can use many winrt functions without prefixes.
Do you think it would be preferable not to use it?

---
In reply to: [424007290](https://github.com/microsoft/react-native-windows/pull/4877#discussion_r424007290) [](ancestors = 424007290)